index.html

```diff
@@ -66,10 +71,10 @@ <h1 class="salutation">
         <h2 class="head">Tonic</h2>
         <h4 class="desktop_show">Tonic</h4>
         <div class="card-detail-box">
-          <ul class="cards-container">
-            <li class="card-detail canopy">CANOPY</li>
-            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
-            <li class="card-detail option"><img src="images/bullets.png" alt="bullet"> 2015</li>
+          <ul class="cards-container popup-container">
+            <li class="card-detail canopy c-js">CANOPY</li>
+            <li class="card-detail option choice"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
+            <li class="card-detail option choice"><img src="images/bullets.png" alt="bullet"> 2015</li>
             <li class="card-detail dcanopy2_face">CANOPY</li>
             <li class="card-detail optiond2_full"><img src="images/bullets.png" alt="bullet"> Back End Dev</li>
             <li class="card-detail optiond2_full"><img src="images/bullets.png" alt="bullet"> 2015</li>
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/132742491-3e8c11af-cf79-4073-9877-bb88907d7231.png)

- [ ] As you can see in the project requirements, you also need to make changes to your HTML code. All the code inside the work section should be removed, and you should generate it using JavaScript using the information you stored in the array. Kindly implement this as it will save you time in the future when updating your Portfolio. You'll only have to add the new project information in the array instead of creating a whole new HTML part for it.

Reply:
I dont quiet understand this because i used what I implemented in the array those are just class names I forgot to rub out in my code otherwise everything from the pop-up being pulled from my Java script

Reply:
@KayLemba Yes, you did generate the pop-up dynamically. The requirement said that you need to do the same to the project cards. So basically your HTML should look something similar to this : 
`<section class="work-section" id="work">`
`</section>`
All the elements inside should be generated dynamicaly

Reply:
I still dont get it  thats the main reason why I asked to call you on zoom. I have seen other peoples work get reviewed and there work looks nothing close to the figma template. I sort of feel you are being unfair on me because i have been working on this project for 3 days now please give me a break